Tasks/MSBuildV1/msbuild.ts

```diff
@@ -59,9 +69,18 @@ async function run() {
             if (msbuildArguments) {
                 buildTool.line(msbuildArguments);
             }
+
+            const startExecTime = new Date().getTime();
             await buildTool.exec();
+            const endExecTime = new Date().getTime();
+
+            const executionTime = (endExecTime - startExecTime) / 1000 // need to convert from milliseconds to seconds
+            telemetry.msbuildExectionTimeSeconds = executionTime;
         }
+
+        emitTelemetry(telemetry);
```

Comment:
Could finally block be more relevant here?

Reply:
Yep, moved this logic to finally block